### PR TITLE
Improve shared status button

### PR DESCRIPTION
**Pull request by marcoambrosini**
* partially fixes --> #41450 

Screenshot 2023-11-14 at 20 09 10

### CODE REVIEW

**Review by jancborchardt — APPROVED** [review]
Looks really nice design-wise! Highlights the shared state while not overly distracting from the file name. :)

File: apps/files/src/components/FileEntry/FileEntryActions.vue [hunk 1]
```diff
@@ -321,3 +322,15 @@ export default Vue.extend({
 	},
 })
 </script>
+
+<style lang="scss" scoped>
+
+:deep(.button-vue--icon-and-text, .files-list__row-action-sharing-status) {
```
Comment (nickvergessen) [inline apps/files/src/components/FileEntry/FileEntryActions.vue:328]: This means one of the libraries has an issue?
Do all apps that use the filepicker from nextcloud/dialogs need to cheat like this?

**Review by emoral435 — APPROVED** [review]
Looks good to me, as we need to get this done by EOD, this maybe can be improved but for now it looks good!

**Comment by jospoortvliet** [comment]
Super nice!

File: apps/files/src/components/FileEntry/FileEntryActions.vue (hunk 1 above)
Comment (marcoambrosini) [reply apps/files/src/components/FileEntry/FileEntryActions.vue:328]: Sorry, In the rush of those days I missed this comment. 
No: this whole set of changes were temporary hacks for files screenshots sake and a less than ideal solutions. I wouldn't have wanted to replicate them in other places.
We talked about using two columns for owner and shared status and doing this properly in the near future.

**Pull request merged**